web_src/less/_repository.less

```diff
@@ -1978,8 +1978,12 @@
     }
 
     #tags-table {
+      .tag {
+        padding: 8px 12px;
+      }
+
       .release-tag-name {
-        font-size: 1.5rem;
+        font-size: 20px;
```

Comment:
Should we stick to `em` units? eg: `1.2em`

Reply:
I wouldn't use `em` except in inline text blocks. It's prone to unexpectedly escalate font size in case of nesting. We should probably introduce some font size helpers later, but I think this is okay for now.